mscore/musescore.cpp

```diff
@@ -8083,6 +8092,7 @@ MuseScoreApplication::CommandLineParseResult MuseScoreApplication::parseCommandL
     parser.addOption(QCommandLineOption({ "E", "install-extension" },
                                         "Install an extension, load soundfont as default unless -e is passed too",
                                         "extension file"));
+    parser.addOption(QCommandLineOption("save-online", "Upload score(s) to their source URL. Replaces existing online score(s)."));
```

Comment:
Maybe add that this is not doing 1st time uploads, just updating existing ones and even that only for the same user?

Reply:
And please also update build/Linux+BSD/mscore.1.in

Reply:
Not sure there's room here, but I added a longer explanation in the manpage.